test/box-py/args.lua

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env tarantool
 
+fio = require('fio')
+
 for i=-1,#arg do
-    print(string.format("arg[%d] => %s", i, arg[i]))
+    local a = arg[i]
+    if string.endswith(arg[i], 'tarantool') then
+        a = fio.basename(arg[i])
+    end
+    print(string.format("arg[%d] => %s", i, a))
```

Comment:
There is the following code in `test/box-py/args.test.py`:

```python
# Replace with the same value for case when builddir inside source dir
sys.stdout.push_filter(re.escape(os.getenv("BUILDDIR")+"/src/tarantool"), "tarantool")
sys.stdout.push_filter(re.escape(os.getenv("BUILDDIR")), "${SOURCEDIR}")
sys.stdout.push_filter(re.escape(os.getenv("SOURCEDIR")+"/src/tarantool"), "tarantool")
sys.stdout.push_filter(re.escape(os.getenv("SOURCEDIR")), "${SOURCEDIR}")
```

Its purpose looks quite similar. I guess that we can solve the problem in one place.

Reply:
Fixed.